en/micro-integrator/docs/references/connectors/importing-connector-to-integration-studio.md

```diff
@@ -0,0 +1,15 @@
+## Importing Connector to Integration Studio Solution Project
+
+1. Open WSO2 Integration Studio and create an **ESB Solution Project**.
+  <img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" />
+
+2. Right click on the project that you created and click on **Add or Remove Connector** -> **Add Connector**.
+<img src="../../assets/img/connectors/add-connector.png" title="Add or Remove Connector" width="800" alt="Add or Remove Connector" />
+
+3. You are directed to the WSO2 Connector Store. 
+
+4. Search for the specific connector and download it to the workspace. 
+<img src="../../assets/img/connectors/search-connector.png" title="Search Connector in the Connector Store" width="800" alt="Search Connector in the Connector Store" />
+
+5. Click **Finish**, and you ESB Solution Project is ready. It will be displayed the downloaded connector in the left side palette. 
```

Comment:
you --> your. 
The connector and its operations will be displayed in the left side palette. You can drag and drop the operations to the design canvas and build your integration logic.

Reply:
We'll add an screenshot for that as well.

Reply:
Done